monai/apps/nnunet/nnunetv2_runner.py

```diff
@@ -473,20 +601,26 @@ def train_parallel(
         **kwargs: Any,
     ) -> None:
         """
+        Create the line command for subprocess call for parallel training.
+        Note: to set the number of GPUs to use, use ``devices_ids`` instead of the `CUDA_VISIBLE_DEVICES`
+        environment variable.
+
         Args:
             configs: configurations that should be trained.
                 default: ("2d", "3d_fullres", "3d_lowres", "3d_cascade_fullres").
-            device_ids: device ids to use for training. default: None.
+            device_ids: a tuple of GPU device IDs to use for the training. Default: None (all available GPUs).
```

Comment:
`list` would work here.

Reply:
Fixed in commit [9af2d74](https://github.com/Project-MONAI/MONAI/pull/6448/commits/9af2d742a8af9bef47cd720b9911d87056f60560)